userguide/plugins.rst

```diff
@@ -130,17 +133,16 @@ page as shown in :numref:`Figure %s <view_installed_plugins_fig>`.
    Viewing Installed Plugins
 
 
-In addition to the :guilabel:`Jail` name, the :guilabel:`Columns` menu controls
-which information is displayed for each installed Plugins entry. Options include
-status, IPv4 address, IPv6 address, installed version, plugin name, release, boot status,
-and collection.
+In addition to the :guilabel:`Jail` name, the :guilabel:`Columns` menu
+controls which information is displayed for each installed Plugins
+entry. Options include status, IPv4 address, IPv6 address, installed
```

Comment:
Let's not list default columns, which can change at any time. The user can see which columns can be displayed in the Columns menu.
```suggestion
entry.
```